### DISCUSSION TITLE
Hyperswitch control center addition to the docker compose

### DISCUSSION THREAD

**Discussion by compgeniuses** [juspay/hyperswitch#4081] [Q&A] [answered]
with the new release and availability completley of the hyperswitch control center. would the docker compose and docker install emthods be updated to inlcude that as well?

**Comment by dvenga** [comment, marked as answer]
@compgeniuses - There are few dependancies for hyperswitch control center to be added to the docker compose. We will check the possibility of getting it added, but this is not on priority as you can do separately run control center image.

**Reply by compgeniuses** [reply]
ok noted
